lib/plugins/aws/package/compile/functions/index.js

```diff
@@ -494,7 +494,12 @@ class AwsCompileFunctions {
             });
         }).then(() => {
           // Include function configuration in version id hash (without the Code part)
-          const properties = _.omit(_.get(functionResource, 'Properties', {}), 'Code');
+          const functionWideProperties = ['ReservedConcurrentExecutions', 'Tags'];
+          const properties = _.omit(
+            _.get(functionResource, 'Properties', {}),
+            'Code',
+            ...functionWideProperties
+          );
```

Comment:
What's the added value of creating `functionWideProperties` variable.

Isn't it simpler to just do `_.omit(..., ['Code', 'ReservedConcurrentExecutions', 'Tags']` ?

Reply:
I thought it would make it more clear as to why these fields are omitted

Reply:
@thewizarodofoz that's a good point , but it'll probably be fine to solve it with simple inline code comment:

```javascript
_.omit(..., [
  'Code',
   // Properties applied to function globally (not specific to version or alias)
  'ReservedConcurrentExecutions', 'Tags'
])
```